components/card/card.js

```diff
@@ -1 +1,25 @@
-export function createCharacterCard() {}
+export function createCharacterCard(name, status, type, occurrences, src) {
+  const cardContainer = document.querySelector('["card-container"]');
+  const newCard = document.createElement("li");
+  newCard.classList.add("card");
+  newCard.innerHTML = `<div class="card__image-container">
+    <img
+      class="card__image"
+      src="${src}"
+      alt="${name}"
+    />
+    <div class="card__image-gradient"></div>
+  </div>
+  <div class="card__content">
+    <h2 class="card__title">${name}</h2>
+    <dl class="card__info">
+      <dt class="card__info-title">Status</dt>
+      <dd class="card__info-description">${status}</dd>
+      <dt class="card__info-title">Type</dt>
+      <dd class="card__info-description">${type}</dd>
+      <dt class="card__info-title">Occurrences</dt>
+      <dd class="card__info-description">${occurrences}</dd>
```

Comment:
💡 Please have a look at the API documents to find out, where to get the information for the occurrences. It is a bit more tricky than the other information.
If you already found it, then just be careful to pass the right arguments to your function when you call it later on 🙂